include/RE/Q/QuestObjectiveStates.h

```diff
@@ -1,14 +0,0 @@
-#pragma once
-
-namespace RE
-{
-	enum class QUEST_OBJECTIVE_STATE
-	{
-		kDormant = 0,
-		kDisplayed = 1,
-		kCompleted = 2,
-		kCompletedDisplayed = 3,
-		kFailed = 4,
-		kFailedDisplayed = 5
-	};
```

Comment:
Why did you change this? Please don't change code for no reason.

Reply:
@Ryan-rsm-McKenzie It was the only quest related enum that wasn't in TESQuest.h so mostly for consistency and reducing the number of tiny unnecessary files. The uppercase to lower was more of subjective change (PascalCase seems to be more commonly used for enums nowadays) but also for consistency with other quest related enum names.

Reply:
This is in its own file to prevent circular dependencies. Please revert this change, and in the future do not change files withy a good reason.

Reply:
@Ryan-rsm-McKenzie reverted